tests/conftest.py

```diff
@@ -31,6 +31,10 @@
 from .small_helpers import SmallHelpers
 from .store_helpers import StoreHelpers
 
+
+FAKE_NOW = dt.datetime(2021, 3, 1)
```

Comment:
Please use fixture instead of global variabe! Also, i remember we made a datetime_now and datetime_yesterday fixtures for cg clean - maybe reuse them? Perhapd youre going to need to move them to higher order of conftest but  we should try and duplicate as little as possible

Reply:
I can't ﬁnd datetime_now in the fixtures, can you point it out for me?

Reply:
https://github.com/Clinical-Genomics/cg/blob/080ba787743b8b3f586ef7352aa9e77154c2e67b/tests/conftest.py#L587